homeassistant/components/schlage/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "schlage",
+  "name": "Schlage",
+  "codeowners": ["@dknowles2"],
+  "config_flow": true,
```

Comment:
For a future PR, you might be able to add DHCP discovery support if the device has a recognizable name and OUI

https://developers.home-assistant.io/docs/creating_integration_manifest/#dhcp

Reply:
That would be great, but I'm not sure if there are recognizable names or OUIs. I'll give it a look though.